tests/functional/test_install.py

```diff
@@ -804,7 +804,7 @@ def test_install_upgrade_editable_depending_on_other_editable(script):
               version='0.1')
     """))
     script.pip('install', '--editable', pkga_path)
-    result = script.pip('list')
+    result = script.pip('list', expect_stderr=True)
     assert "pkga" in result.stdout
```

Comment:
What's this change for? I wouldn't expect any change in behaviour unless ```--columns``` is specified.